src/components/Navigation.tsx

```diff
@@ -752,7 +753,7 @@ const Navigation: FC = () => {
                   <SideNavigationItem>
                     <a
                       className="p-side-navigation__link"
-                      href="https://github.com/canonical/lxd-ui/issues/new"
+                      href={`https://github.com/canonical/lxd-ui/issues/new?labels=bug&title=Bug%20report&body=${encodeURIComponent(getReportBugBody())}`}
```

Comment:
The bug label seems to not be applied. Can you fix it or remove the label parameter?